### DISCUSSION TITLE
Migrate from ElastiCache Redis OSS Serverless Cache to Valkey Serverless Cache

### DISCUSSION THREAD

**Discussion by Shadman19299** [valkey-io/valkey#1325] [General]
**Is there any concrete step by step documentation on this topic of "How to Migrate from ElastiCache Redis OSS Serverless Cache to Valkey Serverless Cache ?"**

**Comment by madolson** [comment]
I think this is the closest thing as of today: https://docs.aws.amazon.com/AmazonElastiCache/latest/dg/VersionManagement.HowTo.cross-engine-upgrade.html.

This is a forum mostly for OSS Valkey, you might want to reach out to AWS support for more info.
